app/controllers/IndexController.scala

```diff
@@ -93,21 +93,22 @@ class IndexController @Inject()(view: IndexView,
     }
   }
 
-  def redirectToAppeals(penaltyId: String, isLPP: Boolean = false, isObligation: Boolean = false,
-                        isAdditional: Boolean = false): Action[AnyContent] = authorise.async {
+  def redirectToAppeals(penaltyId: String, isLPP: Boolean = false, isFindOutHowToAppealLPP: Boolean = false,
```

Comment:
This route is only used by LSP Find Out How to appeal not LPP
```suggestion
  def redirectToAppeals(penaltyId: String, isLPP: Boolean = false, isFindOutHowToAppealLSP: Boolean = false,
```